packages/gatsby-theme-sso-ui-kit/README.md

```diff
@@ -0,0 +1,27 @@
+# commercetools sso-ui-kit plugin
```

Comment:
Moving the documentation to the documentation website  - > note for the separate cleanup PR you intend to do.

Reply:
https://github.com/commercetools/commercetools-docs-kit/issues/1516